proposals/NNNN-android-notifications.md

```diff
@@ -0,0 +1,228 @@
+This is the case if the app is newly installed or updated with SDL support and no other SDL enabled app is installed or connected.
+
+#### 2. The app is connected to an unknown BT device, the app is in the foreground and there is no router service
+
+#### Conditions
+
+- An SDL enabled app is in the foreground on the phone
+- The app never established to register to the connected Bluetooth device
+- On the phone, there is no router service provided by another app 
+
+#### Requirement
+
+1. The app should start the SDL related services as a regular service, without any foreground notification.
+2. The app should keep the services up and running for the time the app is in the foreground.
+3. The app should stop the services when entering the background.
+
+#### Comments
+
+This is required to avoid showing up any SDL foreground notifications when connected to the non-SDL system.
+
+#### 3. The app is in the foreground and established SDL session with a connected system
+
+#### Conditions
+
+- An SDL enabled app is in the foreground on the phone
+- The app never established to register to the connected Bluetooth device
+- On the phone, there is no router service provided by another app 
+- The app got connected and registered to the head unit
+
+#### Requirements
+
+1. The app should store that information
+2. The router service shows foreground notifications to the user.
+
+#### Comments
+
+This is the end of the "first time" guide storing the information of the user owning an SDL capable device
+
+#### 4. Router service with the app in background for known SDL BT devices
+
+#### Conditions
+
+- An SDL enabled app is not in the foreground on the phone
+- The app established to register to the connected Bluetooth device
+- On the phone, there is no router service provided by another app 
+
+#### Requirements
+
+The app should start the router service as a foreground service including a notification.
+
+#### Comments
+
+As the app knows that the device supports SDL it can start the router service and keep it active/foregrounded
+
+#### 5. The app connects to router service when in background
+
+#### Conditions
+
+- An SDL enabled app is in the foreground on the phone
+- The app never established to register to the connected Bluetooth device
+- On the phone, another app has created the router service
+
+#### Requirements
+
+1. The app's proxy should connect to the router service
+2. The router service should tell the app if the connected BT device is known to support SDL
+
+#### Comments
+
+This behavior can exist if multiple SDL apps are installed on users device.
+
+## Potential downsides
+
+The downside is the reduce experience for the first time use as the phone must be unlocked and the app must be started. However, this proposal reduces the intrusion of SDL foreground services.
+
+## Impact on existing code
+
+Changes to the library can be high. It's likely this to require a major version change as it could change critical areas of multiplexing.
```

Comment:
```suggestion
Changes to the library can be high. It's likely that this will require a major version change as it could change critical areas of multiplexing.
```
I did not see these changes listed in the `Proposed solution` section.  Please make sure these are included, thanks!